spec/std/yaml/yaml_pull_parser_spec.cr

```diff
@@ -95,5 +106,11 @@ module YAML
         end
       end
     end
+
+    assert_raw %(hello)
+    assert_raw %("hello"), %(hello)
+    assert_raw %(["hello"])
+    assert_raw %(["hello","world"])
+    assert_raw %({"hello":"world"})
```

Comment:
That's fine, I'd just like to make sure stuff like

```yml
foo: &foo
  - "foo"
  - false
- foo: 1
  bar:
    <<: *foo 
- "hello world\n\nyay"
```

is converted correctly :)